lms/static/sass/shared-v2/_header.scss

```diff
@@ -0,0 +1,80 @@
+// edX theme: Global Header
+// ====================
+
+.header-global {
+  @extend %ui-depth1;
+  @include box-sizing(border-box);
+  position: relative;
+  width: 100%;
+  border-bottom: 4px solid $courseware-border-bottom-color;
```

Comment:
I think for border widths it's fine (this was taken from the edx-platform Sass). Because it's not really reused I'm not sure of the value in making it a variable.